### DISCUSSION TITLE
email with sceewlog

### DISCUSSION THREAD

**Discussion by dufangqing** [SED-EEW/SED-EEW-SeisComP-contributions#56] [Q&A] [answered]
Hi there,

I am trying to enable the email notifications in sceewlog with the following settings but I cannot recieve emails:

<img width="993" alt="image" src="https://github.com/SED-EEW/SED-EEW-SeisComP-contributions/assets/101325219/b37693fa-43a1-4727-9b79-1c748bbe0eb3">

I also tried with the default port and there is no luck. I can confirm the credential file is set up correctly, do you have any ideas about what might go wrong?

Thanks a lot,
Du

**Comment by FMassin** [comment, marked as answer]
Hello

Here's my usual config style with gmail:
```bash
email.activate = true
email.magThresh  =  0.0
email.usetls =  true
email.usessl = false
email.port  =  587
email.authenticate  =  true
email.smtpserver  =  smtp.gmail.com
email.credentials  =  /home/sysop/.seiscomp/.useremail
email.senderaddress  =  ****@gmail.com
email.subject  =  "EEW /"
email.host  =  ***.***.ethz.ch
email.recipients  =  fmassin@...
```

And my `/home/sysop/.seiscomp/.useremail`:
```text  
username=*****@gmail.com
password=*****
```

Check your gmail account config to make sure it allows using email.mime.text.MIMEText() : https://pythonaddict.com/how-to-generate-google-application-password/

Let me know if that helps!

Cheers

Fred

**Reply by dufangqing** [reply]
Thank you Fred! Problem solved after I used the app psw for authentication.
